scripts/index.js

```diff
@@ -0,0 +1,70 @@
+console.log("hellooo!")
+
+//store current temp in a variable
+
+const state = {
+    //color: "red",
+    tempCount : 70,
+};
+
+
+//increase temp
+const increaseTemp= () => {
+    state.tempCount +=1
+    const tempUp = document.querySelector("#temp")
+    tempUp.textContent = `${state.tempCount}`
+    //updateTempColor();
+};
+
+const decreaseTemp= () => {
+    state.tempCount -=1
+    const tempDown = document.querySelector("#temp")
+    tempDown.textContent = `${state.tempCount}`
+    //updateTempColor();
+};
```

Comment:
Just note that you can combine these functions and use a parameter.

```suggestion
//increase temp
const changeTemp= (increment) => {
    state.tempCount +=increment
    const tempUp = document.querySelector("#temp")
    tempUp.textContent = `${state.tempCount}`
    //updateTempColor();
};
```

Then you can pass these in as event handlers

`upButton.addEventListener("click",() => changeTemp(1));`

`downButton.addEventListener("click",() => changeTemp( -1 ));`